specs/src/specs/params.md

```diff
@@ -0,0 +1,58 @@
+| consensus.block.MaxBytes | 1.88MiB | Governance parameter for the maximum size of the protobuf encoded block. | True |
+| consensus.block.MaxGas | -1 | Maximum gas allowed per block (-1 is infinite). | True |
+| consensus.block.TimeIotaMs | 1000 | Minimum time added to the time in the header each block. | False |
+| consensus.evidence.MaxAgeNumBlocks | 100000 | The maximum number of blocks before evidence is considered invalid. This value will stop CometBFT from pruning block data. | True |
+| consensus.evidence.MaxAgeDuration | 1814400000000000 (21 days) | The maximum age of evidence before it is considered invalid in nanoseconds. This value should be identical to the unbonding period. | True |
+| consensus.evidence.MaxBytes | 1MiB | Maximum size in bytes used by evidence in a given block. | True |
+| consensus.validator.PubKeyTypes | Ed25519 | The type of public key used by validators. | False |
+| consensus.Version.AppVersion | 1 | Determines protocol rules used for a given height. Incremented by the application upon an upgrade. | False |
+| distribution.communitytax | 2.0% | Percentage of the inflation sent to the community pool. | True |
+| distribution.WithdrawAddrEnabled | true | Enables delegators to withdraw funds to a different address. | True |
+| distribution.BaseProposerReward | 0 | Reward in the mint demonination for proposing a block. | True |
+| distribution.BonusProposerReward | 0 | Extra reward in the mint denomination for proposers based on the voting power included in the commit. | True |
+| gov.DepositParams.MinDeposit | 1000000000utia (1000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
+| gov.DepositParams.MaxDepositPeriod | 172800 (2 days) | Maximum period for token holders to deposit on a proposal in seconds. | True |
+| gov.VotingParams.VotingPeriod | 172800 (2 days) | Duration of the voting period in seconds. | True |
+| gov.TallyParams.Quorum | 33.4% | Minimum percentage of total stake needed to vote for a result to be considered valid. | True |
+| gov.TallyParams.Threshold | 50.0% | Minimum proportion of Yes votes for proposal to pass. | True |
+| gov.TallyParams.VetoThreshold | 33.4% | Minimum value of Veto votes to Total votes ratio for proposal to be vetoed. | True |
+| ibc.ClientGenesis.AllowedClients | []string{"06-solomachine", "07-tendermint"} | List of allowed IBC light clients. | True |
+| ibc.ConnectionGenesis.MaxExpectedTimePerBlock | 7500000000000 (75 seconds) | Maximum expected time per block in nanoseconds under normal operation. | True |
+| ibc.Transfer.SendEnabled | true | Enable sending tokens via IBC. | True |
+| ibc.Transfer.ReceiveEnabled | true | Enable receiving tokens via IBC. | True |
+| slashing.SignedBlocksWindow | 5000 | The range of blocks used to count for downtime. | True |
+| slashing.MinSignedPerWindow | 5 | Minumum signatures in the block. | True |
```

Comment:
> Minumum signatures in the block.

The description here should be changed to sth more descriptive: Minumum number of blocks signed in the `SignedBlocksWindow`. Or sth like that?

Reply:
cc @rootulp @evan-forbes

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/2469